### PR TITLE
EP-92 Burendo handbook hosting

### PR DESCRIPTION
**Pull request by connoravo**

### CODE REVIEW

File: .github/workflows/tf-merge.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ jobs:
         id: bootstrap
         run: |
           python bootstrap.py > ${{ github.workspace }}/${{ github.run_id }}-${{ github.run_number }}.log
+          sed -i "s/Administrator/ci/" variables.tf
```
Comment (ChrisScottThomas) [inline .github/workflows/tf-merge.yml:43]: is this correct?  I thought you'd placed the TF files inside `terraform/`?

File: .github/workflows/tf-merge.yml [hunk 2]
```diff
@@ -73,6 +74,35 @@ jobs:
...
+        run: npm run build
+        working-directory: ./burendo-handbook
+        continue-on-error: false
+
+      - name: "Copy build output to S3"
+        id: copy-to-s3
+        run: |
+          cd terraform
+          output=$(terraform output -json)
+          s3_bucket=$(echo $output | jq -r .burendo_handbook_s3.value)
```
Comment (ChrisScottThomas) [inline .github/workflows/tf-merge.yml:100]: I thought this didn't work and needed to switch to env vars?

**Review by ChrisScottThomas — COMMENTED** [review]
Minor comments

File: .github/workflows/tf-merge.yml (hunk 2 above)
Comment (connoravo) [reply .github/workflows/tf-merge.yml:100]: This is TF-MERGE. I haven't tweaked this - next ticket is to productionise.

File: .github/workflows/tf-merge.yml (hunk 1 above)
Comment (connoravo) [reply .github/workflows/tf-merge.yml:43]: TF-MERGE. Will solve in next ticket when productionising.

**Review by ChrisScottThomas — APPROVED** [review]

**Pull request merged**